recipe/meta.yaml

```
@@ -27,13 +27,15 @@ requirements:
     - cloudpickle >=1.6.0,<=2.0.0
     - snowflake-connector-python >=2.7.12,<4.0.0
```

```suggestion
    - snowflake-connector-python >=3.0.4,<4.0.0
```
https://github.com/snowflakedb/snowpark-python/blob/v1.6.1/setup.py#L13